lib/commands/general.js

```diff
@@ -42,6 +42,10 @@ commands.mobileIsToastVisible = async function (opts = {}) {
   });
 };
 
+commands.mobileBackdoor = async function (opts = {}) {
+  return await this.espresso.jwproxy.command(`/appium/execute_mobile/backdoor`, 'POST', {opts});
```

Comment:
I'd rather add a verification for missing arguments, which are mandatory

Reply:
I have added verification of mandatory arguments on the server side. It will raise an exception in case it is missing. Do you think its needed here as well?

Reply:
yes. This way you don't need to study the server code in order to understand what is necessary to make the stuff working.
Also, it might be handy to have some short docstring here with shallow description/usage example since the method itself is quite complicated

Reply:
Added